OpenOversight/app/main/views.py

```diff
@@ -769,6 +769,8 @@ def edit_department(department_id: int):
         department.name = form.name.data
         department.short_name = form.short_name.data
         department.state = form.state.data
+        department.last_updated_by = int(current_user.get_id())
```

Comment:
During testing, this kept being returned as a string, so I had to cast it as an int. My hope is that it's at worst redundant.

Reply:
This is really strange! Would we want to dig into why that's happening?

Reply:
I just took a look and it looks like this is because we are using `current_user.get_id()`, which is a Flask-Login function which is required to return a string. You should be able to use `current_user.id` to get an int id instead

https://flask-login.readthedocs.io/en/latest/#your-user-class

Reply:
Excuse me, I didn't see this question. I'll make the update now.

Reply:
Fixed!